### DISCUSSION TITLE
transfering RAW images through FTP

### DISCUSSION THREAD

**Discussion by ericcatta** [aaronwmorris/indi-allsky#1265] [Q&A]
I am currently trying to transfer through ftp the images to a NAS. Everything works fine: I get all the videos, panoramas,... that I want. The only problem that I have is that I would like to transfer automatically the raw data (.tif) files to my NAS but I can't figure out how to do it. is there a way of doing it?
thank you in advantage for your help

**Comment by ericcatta** [comment]
P.S.: i curently am transferring the .jpeg images perfectly

**Reply by aaronwmorris** [reply]
I will have to add the functions to perform file transfers for RAW files.

**Reply by ericcatta** [reply]
ok, i understand. can you think of any other solution I could try in the meanwhile?

**Reply by aaronwmorris** [reply]
I will work on it today

**Comment by aaronwmorris** [comment]
Merged #1267 with functionality to transfer RAW and FITS files.
